src/Compilers/CSharp/Portable/FlowAnalysis/NullableWalker.cs

```diff
@@ -338,7 +358,8 @@ protected override ImmutableArray<PendingBranch> Scan(ref bool badRegion)
             try
             {
                 bool badRegion = false;
-                ImmutableArray<PendingBranch> returns = walker.Analyze(ref badRegion);
+                Optional<LocalState> initialLocalState = initialState is null ? default : new Optional<LocalState>(initialState.VariableStates);
```

Comment:
>default [](start = 80, length = 7)

nit: null?

Reply:
No, `Optional<LocalState>` is a value type.

---
In reply to: [267583087](https://github.com/dotnet/roslyn/pull/34300#discussion_r267583087) [](ancestors = 267583087)